pkg/apis/v1alpha1/types.go

```diff
@@ -68,15 +68,20 @@ type PlacementRequestResult string
 type PlacementRequestSpec struct {
 	// Policy indicates the relationship between the bindings in a
 	// placement request.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Enum=Lenient;AllOrNothing
```

Comment:
We could allow "" that defaults to "Lenient" (which can change over time).

Reply:
I don't like that a lot. I mean, we would need to document this and we would be defining a default we are not sure it is the one we really want. I would rather leave this as is, we can always define a default later on.